crates/bevy_ecs/src/reflect/component.rs

```diff
@@ -106,9 +109,9 @@ pub struct ReflectComponentFns {
     /// Function pointer implementing [`ReflectComponent::contains()`].
     pub contains: fn(EntityRef) -> bool,
```

Comment:
Nit: For completeness, could we update this method as well?

Reply:
Done.